extension/js/common/composer.ts

```diff
@@ -686,10 +686,8 @@ export class Composer {
     this.S.cached('send_btn_note').text('');
     this.S.cached('send_btn').removeAttr('title');
     const wasPreviouslyVisible = this.S.cached('password_or_pubkey').css('display') === 'table-row';
-    if (!$('.recipients span').length) {
-      this.hideMsgPwdUi();
-      this.S.cached('send_btn').removeClass('gray').addClass('green');
-    } else if (this.S.cached('icon_sign').is('.active')) {
+    this.hideMsgPwdUi();
```

Comment:
Here you're always hiding it, and then maybe re-showing later, which cannot be a good thing

Reply:
Yes, I also wasn't sure about it, 

Thanks for your answer in the issue